release-notes.py

```diff
@@ -9,11 +10,20 @@
 
 xml_version = pomxml.getElementsByTagName('version')
 xml_name = pomxml.getElementsByTagName('name')
+xml_message = pomxml.getElementsByTagName('message')
+xml_use_ticket = pomxml.getElementsByTagName('use-ticket')
 
 app_ver = xml_version[0].firstChild.data
 app_name = xml_name[0].firstChild.data
+app_msg = ""
+app_use_tckt = ""
```

Comment:
This variable naming looks strange on line 14 you use ticket in name, so I would keep it similar, so you will see its somehow connected.